src/stores/events/EventsDataStore.ts

```diff
@@ -296,6 +306,7 @@ export default class EventsDataStore {
 				if (
 					rootNode &&
 					(isRootEvent(rootNode) || rootNode.isUnknown) &&
```

Comment:
now it gets a little bit simplier
we want to add to rootEventIds only if its isRootEvent and `!this.rootEventIds.includes(rootNode.eventId)`